pages/index.js

```diff
@@ -12,33 +12,61 @@ export default function Home() {
+        </p>
+        <p>
+          To request a repair for a <strong>communal area</strong>, please see customer services contact <TextLink href="https://www.lincoln.gov.uk/contact-1/get-touch">details page</TextLink>.
+        </p>
+        <Details summary="Which areas are communal?" className="govuk-!-margin-top-6" data-testid="landing-page-communal-prompt">
+          <div data-testid='landing-page-communal-info'>
+            <p>
+              Communal repairs are usually in areas that people share.
+            </p>
+            <p>
+              They can include:
+            </p>
+            <ul>
+              <li>repairs to door entry systems</li>
+              <li>lock repairs to communal doors</li>
+              <li>lighting repairs to shared areas</li>
+              <li>glazing repairs to shared doors or stairway windows</li>
+              <li>roof and gutter repairs</li>
+              <li>structure and external walls to your block</li>
+            </ul>
+          </div>
+        </Details>
+        <br/>
+        <WarningText testid="landing-page-gas-warning" className="govuk-!-margin-top-4">
             If you suspect you have a gas leak, you must report it immediately
             to the Gas Emergency Service on 0800 111 999 or via textphone
             (minicom) on 0800 371 787
         </WarningText>
-        <WarningText testid="landing-page-emergency-warning" className="govuk-!-margin-top-4">
-            For other emergency repairs, please see customer services {contactDetailsPageLink}.
+        <WarningText testid="landing-page-emergency-warning"  className="govuk-!-margin-top-4">
+          For other emergency repairs, please see customer services <TextLink href="https://www.lincoln.gov.uk/contact-1/get-touch">contact details page</TextLink>.
         </WarningText>
         <Details summary="What is an emergency?" className="govuk-!-margin-top-6" testid="landing-page-emergency-prompt">
           <div data-testid='landing-page-emergency-info'>
             <p>
-                An emergency is defined as something which could cause danger to
-                someone’s health or safety or cause serious damage and destruction
-                to property.
+              An emergency is defined as something which could cause danger to
+              someone’s health or safety or cause serious damage and destruction
+              to property.
             </p>
             <p>
-                Emergency Out of Hours Repairs, please call <strong>01522 534747</strong>
+              Emergency Out of Hours Repairs, please call <strong>01522 534747</strong>
             </p>
             <p>
-                For emergencies in normal hours, please call <strong>01522 873333</strong>
+              For emergencies in normal hours, please call <strong>01522 873333</strong>
             </p>
             <p>
-                Opening times:
+              Opening times:
```

Comment:
Opening hours in the prototype are different to the MVP:
Differences:

Prototype has Wednesday 10am - 4.30pm and MVP has Wednesday 9am - 4.30pm
Prototype has Thursday 9am - 5pm and MVP has Thursday 10am - 5pm